docs/advanced-features/custom-document.md

```diff
@@ -71,7 +71,9 @@ class MyDocument extends Document {
     // Run the parent `getInitialProps`, it now includes the custom `renderPage`
     const initialProps = await Document.getInitialProps(ctx)
 
-    return initialProps
+    return (
```

Comment:
Since we aren't showing this variable being used we could change this to `return Document.getInitialProps(ctx)`

Reply:
Then is what I found intended, not a mistake in the ``next.js documentation``?

Reply:
This is not a mistake, it's just a suggestion from your WebStorm that says you can skip variable assignment and return `initialProps` directly on the same line of `Document.getInitialProps()` call

Reply:
Oh, I see.. thank you for your feedback.